ci/gpu/build.sh

```diff
@@ -41,7 +41,14 @@ nvidia-smi
 
 logger "Activate conda env..."
 source activate gdf
-conda install "rmm=$MINOR_VERSION.*" "nvstrings=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL"
+conda install "rmm=$MINOR_VERSION.*" "nvstrings=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
+              "dask>=2.0" "distributed>=2.0"
```

Comment:
You might want `dask-core` here rather than `dask`.  Dask is a metapackage that includes things like numpy, pandas, distributed, and bokeh

Reply:
@mrocklin tried with `dask-core` and had issues with `partd` missing and I assume other packages, so would rather just get this in and we can refine from there.